assets/src/edit-story/components/inspector/design/designInspector.js

```diff
@@ -26,21 +26,23 @@ import { LayerPanel } from '../../panels';
 import DesignPanels from './designPanels';
 
 const Wrapper = styled.div`
-  height: 100%;
+  height: calc(100% - 40px);
+  width: 100%;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  position: absolute;
```

Comment:
Wouldn't `position:absolute` and `flex-grow: 1` conflict each other? Is it necessary?